Compiler/Compiler.py

```diff
@@ -64,9 +64,9 @@ def compile_global_variable_definition(self):
                 code = (code + '>') * get_variable_size(variable)  # advance to after this variable
                 return code
             elif self.parser.current_token().type == Token.ASSIGN and self.parser.current_token().data == "=":
-                # array definition and initialization - INT ID (LBRACK NUM RBRACK)+ ASSIGN (LBRACE ... RBRACE)+ SEMICOLON
+                # array definition and initialization - INT ID (LBRACK NUM RBRACK)+ ASSIGN ((LBRACE ... RBRACE)+|STRING) SEMICOLON
                 self.parser.advance_token()  # skip ASSIGN
-                self.parser.check_current_tokens_are([Token.LBRACE])
+                assert self.parser.current_token().type in [Token.LBRACE, Token.STRING]
```

Comment:
Assert is used as sanity check for the compiler developer
Here we need to inform the user of the syntax error they made
For example, if my code is `char a[10] = 1;` then the output is just AssertionError with no indication where
Please change it back to check_current_tokens_are - this function informs the user where the error is.
Might need to do "if token is LBRACE then x, elif token is STRING then y", and then use this function with different parameters

Reply:
The check_current_tokens_are only checks one token.

Would this be okay?
```
if current_token.type in [...]:
    ...
else:
    raise BFSyntaxError("Expected LBRACE or STRING at '%s'" % current_token)
```

Reply:
check_current_tokens_are receives a list of tokens and checks all of them exist and in order
In this case, it is not exactly what we want because the next token can be either LBRACE or STRING
Your code snippet is a good way to implement this, so yes :)